configs/manifests/alerting-persist.yml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
What would we loose if we don't use persistent volume?

Reply:
It is needed for saving list of chat that the bot added to them. see [here.](https://github.com/metalmatze/alertmanager-bot/blob/11438e166eac991c6540e10dcfdcabcb2006e349/pkg/telegram/chats.go)

Reply:
ok lets make it really small as I don't think it will use much space. something like 100MB